apps/design/frontend/src/ballot_order_info_screen.tsx

```diff
@@ -266,7 +266,7 @@ export function BallotOrderInfoScreen(): JSX.Element | null {
   return (
     <ElectionNavScreen electionId={electionId}>
       <MainHeader>
-        <H1>Ballot Order Info</H1>
+        <H1>Order Ballots</H1>
```

Comment:
Note that I'm not renaming all the underlying components, because I don't want to introduce merge conflicts right now. We can clean this up after the fact: https://github.com/votingworks/vxsuite/issues/5928